components/form/formitems/overview.md

```diff
@@ -75,12 +76,40 @@ The `FormItem` tag exposes the following parameters which you can use to customi
 }
 ````
 
->caption The result from the code snippet above
+## Hybrid Form - Combination of manually defined and autogenerated fields
```

Comment:
This heading is a bit long. How about "Add Form Fields to Autogenerated Ones" or "Use FormItems Together with Autogenerated Ones"